charts/openinwoner/templates/configmap.yaml

```
@@ -83,3 +83,8 @@ data:
   {{- if .Values.settings.uwsgi.httpKeepalive }}
   UWSGI_HTTP_KEEPALIVE: {{ if .Values.settings.uwsgi.httpKeepalive }}"1"{{ else }}"0"{{ end }}
   {{- end }}
+  {{- if .Values.settings.allowCustomJs }}
+  ALLOW_CUSTOM_JS: "True"
+  {{- else }}
+  ALLOW_CUSTOM_JS: "False"
+  {{- end }}
```

```suggestion
  ALLOW_CUSTOM_JS:  {{ if .Values.settings.allowCustomJs }}"True"{{ else }}"False"{{ end }}
```
Minor, but for consistency, I see that the other bools are set like this?